src/app/project/ontology/default-data/default-properties.ts

```diff
@@ -112,7 +124,7 @@ export class DefaultProperties {
                 //     description: 'A period of time between two dates',
                 //     subPropOf: Constants.HasValue,
                 //     objectType: Constants.DateValue,
-                //     guiEle: Constants.SalsahGui + Constants.HashDelimiter + 'Date',
+                //     guiEle: Constants.GuiDatePicker',
```

Comment:
is this commented out code going to be needed in the future or can we remove it?

Reply:
I thought there is or there will be a difference between a date and a date period. But in our case you can always create a date period from a "normal" date property. So, there's no additional date period property needed. I can remove this section.